projectforum/projects/templates/project_detail.html

```diff
@@ -15,6 +15,12 @@
     <div class='project_header'>
         <h4 class='project_status'>{{ project.get_status_display }}</h4>
         <div class='title_tab'>
+	        {% if logged_in and user != project.owner %}
+			    <a href="#" id="bookmark" title="Bookmark this project" data-projectid="{{ project.id }}">&nbsp;</a>
```

Comment:
Can we not use the id bookmark? it feels like something that could be accidentally used. Maybe prefix it with project or something.